bears/php/PHPCodeBeautifierBear.py

```diff
@@ -0,0 +1,177 @@
+                        allow_multiple_statements_per_line: bool=False,
+                        force_lower_case_keywords: bool=True,
+                        force_lower_case_constants: bool=True,
+                        blank_line_after_namespace_declaration: bool=True,
+                        check_use_blocks: bool=True,
+                        check_class_declaration: bool=True,
+                        check_property_declaration: bool=True,
+                        force_scope_modifier_on_method: bool=True,
+                        function_declaration_argument_spacing: int=1,
+                        allow_multiline_function_declaration: bool=True):
+        """
+        :param max_line_length:
+            Maximum number of characters for a line.
+        :param line_ending_character:
+            Checks that end of line characters correspond to the one provided.
+        :param indent_size:
+            Number of spaces per indentation level.
+        :param use_spaces:
+            True if spaces are to be used instead of tabs.
+        :param allow_multiple_statements_per_line:
+            Allows having multiple statements on one line.
+        :param force_lower_case_keyword:
+            Checks that ``PHP`` keywords are lowercase.
+        :param force_lower_case_constant:
+            Checks that all uses of ``true``, ``false`` and ``null`` are
+            lowercase.
+        :param blank_line_after_namespace_declaration:
+            Ensures that there is a blank line after a namespace declaration.
+        :param check_use_blocks:
+            Ensures that there is one blank line after a ``use`` block,
+            that there is only one use block per line, and that all ``use``
+            declaration are done after namespaces declaration.
+        :param check_class_declaration:
+            Ensures that ``extends`` and ``implements`` keywords are declared
+            on the same line as the class name, that the opening brace for a
+            class is on the next line, and that the closing brace for a class
+            is on the next line after the body. Allows splitting implements
+            list accross multiple lines.
+        :param check_property_declaration:
+            Ensures that visibility is declared on all properties, that the
+            ``var`` keyword is not used to declare a property, that there is
+            not more that one property declared on a line, that properties are
+            not prefixed with an underscore.
+        :param force_scope_modifier_on_method:
+            Verifies that class methods have scope modifiers.
+        :param function_declaration_argument_spacing:
+            Number of spaces between arguments in function declaration.
+        :param allow_multiline_function_declaration:
+            Allows argument lists to be split accross multiple lines correctly
+            indented.
+        """
+        rules_map = {'Generic.WhiteSpace.DisallowTabIndent':
```

Comment:
the indentation does not seem right in this dict, can you keep the `key: value` in one line

Reply:
But than it would exceed max_line_length limit